css/layout.css

```diff
@@ -0,0 +1,29 @@
+div.carousel {
+  margin-left: 10%;
+  margin-right: 10%;
+}
+
+header {
+  margin-bottom: 40px;
+}
+
+.hidden {
+  display: none !important;
+}
+
+.entry {
+  margin-left: 10%;
+  margin-right: 10%;
+}
```

Comment:
Please make sure you're using the right sizing units for the CSS properties you're adjusting: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css